README.md

```diff
@@ -253,6 +253,15 @@ window_start_time = Time.now - 24 * 3600
 bundle exec ruby lib/zendesk-ticket-deduplicator.rb
 ```
 
+##### Deploy the pipeline
+
+```
+fly  -t cd-autom8 set-pipeline -p zendesk-deduplicator -c ci/zendesk_de-duplicator.yml
+```
+
+##### Important Note
+
+This pipeline should only run 6 hourly in order to mesh with the manual processes carried out by the support team. If updating the code, please pause the pipeline to ensure it does not run between the 6 hour windows.
```

Comment:
Can you add the "why" here ... ie "running outside of the window may cause unexpecting moving of tickets around during the hours people are working on them" .... so that future people who _need_ to make a change know what the impact will be.

Reply:
Thanks @chrisfarms embodied your suggestion.